render-student.js

```diff
@@ -1,14 +1,26 @@
+function renderData(students) {
+    for (let student of students) {
+        if (student.photo === null) {
+            student.photo = "https://oldough.com/static/images/avatar.png";
+        }
+        if (student.bio === null) {
+            student.bio = "This is my bio...";
+        }
+    }
+}
+
 // sto2html = student object to html
 function renderStudent(student = {}) {
 	return `<div class="student">
     <h4>${student.name}</h4>
     <div class="photo"><img src='${student.photo}'>
```

Comment:
The solution should be in here, like:
```suggestion
     <div class="photo"><img src='${student.photo || "the fallback URL"}'>
```